<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWinClass</class>
 <widget class="QWidget" name="PreferencesWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>696</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="miscTab">
    <attribute name="title">
     <string>Look and Feel</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>151</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Theme</string>
     </property>
     <widget class="QRadioButton" name="classicRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Classic</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="modernRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Modern</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="pswDock">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Show PSW in a docked window</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="MipsMeterBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>151</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Mips Metering</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="mipsLedButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LED display</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="mipsGaugeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gauge</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>651</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Font Definitions</string>
     </property>
     <widget class="QComboBox" name="fontNameLog">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="fontSizeLog">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="boldLog">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicLog">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Italic</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Window</string>
      </property>
     </widget>
     <widget class="QComboBox" name="fontNameRegs">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="fontNamePsw">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="fontNameCommand">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="fontSizeRegs">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="fontSizePsw">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="fontSizeCommand">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="boldRegs">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="boldPsw">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="boldCommand">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicRegs">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicPsw">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicCommand">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Registers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PSW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Command Line</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration Edit</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicConfiguration">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="fontSizeConfiguration">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="boldConfiguration">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>79</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="fontNameConfiguration">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="directoriesTab">
    <attribute name="title">
     <string>Directories</string>
    </attribute>
    <widget class="QLineEdit" name="logsDirLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hercule's Directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Configuration Directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs Directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hercDirLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is the directory which contains the hercules executable. If hercules is on your shell's path, you can leave this field empty</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="configDirLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="configDirButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logsDirButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hercDirButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loggingTab">
    <attribute name="title">
     <string>Logging</string>
    </attribute>
    <widget class="QCheckBox" name="logTimestampCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Append time to each log line</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="autosaveLogCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Autosave log</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="splitLogCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Split Hercules and O/S log  (requires restart)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="memLines">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>5000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum lines to keep for scrollback</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
